src/Commands/InstallCommand.php

```diff
@@ -62,4 +65,47 @@ protected function resolveBaseStructures(): array
 
         return $structures;
     }
+
+    protected function resolveInfrastructurePath(): string
+    {
+        $path = base_path() . '/' . config(key: 'diamond.base_directory') . config(key: 'diamond.structures.infrastructure') . '/Laravel/Providers';
+
+        return $path;
+    }
+
+    protected function resolveNamespace(): string
+    {
+        $namespace = config(key: 'diamond.structures.infrastructure') . '\\Laravel\\Providers';
+
+        return $namespace;
+    }
+
+    protected function resolveRefactor(): void
+    {
+        if (! $this->option('skip-refactor')) {
+            $filesystem = new Filesystem;
+            $providerPath = app_path(path: 'Providers');
```

Comment:
since this variable isn't get use in anywhere you can put to the argument directly